tox.ini

```diff
@@ -41,14 +42,15 @@ deps =
     dj17: Django>=1.7,<1.8
     dj18: Django>=1.8,<1.9
     dj19: Django>=1.9,<1.10
+    dj110: Django>=1.10,<1.11
     djdev: https://github.com/django/django/archive/master.tar.gz
 
-    py27-dj{17,18,19}: django-oauth-plus==2.2.8
-    py27-dj{17,18,19,dev}: python-digest
-    py27-dj{17,18,19,dev}: oauth2
-    py27-dj{17,18,19,dev}: pysqlite==2.7.0
-    py34-dj{17,18,19,dev}: python3-digest>=1.8b4
-    dj{17,18,19,dev}: -r{toxinidir}/tests/requirements.txt
+    py27-dj{17,18,19,110}: django-oauth-plus==2.2.8
```

Comment:
I think those oauth tests only run if django-oauth-plus is installed, and before this line was changed we weren't installing it on dev, probably due to the failure. I'd try bumping the version or removing it for dj110.